systemd-swap.service

```diff
@@ -25,4 +25,4 @@ RestrictAddressFamilies=AF_UNIX AF_INET AF_INET6 AF_NETLINK
 SystemCallFilter=@system-service @swap @module
 
 [Install]
-WantedBy=sysinit.target
+WantedBy=multi-user.target
```

Comment:
No, whilst this might not make a difference on most if not all systems, semantically this is wrong. Looking at the output of `systemctl list dependencies multi-user.target` we'll see a lot of dependencies on `sysinit.target` by daemons, and this is where they belong. depending on `multi-user.target` is to my information something that user facing services should do (although there are exceptions such as NetworkManager.service), and we should just hold ourselves to that. You can also read more [here](https://unix.stackexchange.com/questions/556923/when-should-a-systemd-unit-file-use-sysinit-target-vs-multi-user-target).

Reply:
Since we rely on `/var` which could be on another device, I don't think the `sysinit.target` is the correct choice. At least not without being more explicit about the dependencies.
See `man 7 systemd.special`.

Reply:
Ah yes, this is somewhat awkward but I think you might be correct 😅.